src/parsing/index.ts

```diff
@@ -118,3 +119,16 @@ export function parseGenericSql(queryBeforeCursor: string, queryAfterCursor: str
     let parser = genericAutocompleteParser as Parser;
     return parser.parseSql(queryBeforeCursor, queryAfterCursor);
 }
+
+export function parseGenericSqlWithoutCursor(queryBeforeCursor: string): ParseResult {
```

Comment:
This parameter should be renamed to `query`. Same for `parsePostgreSqlWithoutCursor`